README.rst

```diff
@@ -535,6 +535,25 @@ example, an interactive mode would be enabled in this run::
       $ TMT_PLUGIN_DISCOVER_FMF_VERBOSE=2 tmt run -a discover -h fmf ...
 
 
+Build Variables
```

Comment:
How much is `Contributing.rst` used? Would it make sense to put it there?

Reply:
It could go to the [contribute/docs](https://tmt.readthedocs.io/en/latest/contribute.html#docs) section as well. Hm, and perhaps it makes a little bit more sense there? @happz, thoughts?

Reply:
No preferences. I went to `README.rst` as that's where variables live now, that's all.

Reply:
Ok, moved to `contribute` in dad9e4ca.